package.json

```diff
@@ -21,7 +21,7 @@
   "homepage": "https://github.com/lennym/timeliner#readme",
   "dependencies": {
     "JSONStream": "^1.2.1",
-    "babar": "0.0.3",
+    "babar": "https://github.com/cubelets/babar#feature/configurable-viewport",
```

Comment:
Yes. I put a comment on the PR, but it's only a minor thing, and not related to the code.